frontend/packages/signless-transactions/src/components/create-session-modal/create-session-modal.tsx

```diff
@@ -87,39 +87,46 @@ function CreateSessionModal({ close }: Props) {
 
   return (
     <>
-      <Modal heading="Create Signless Session" close={close}>
+      <Modal heading="Enable Signless Session" close={close}>
         <ul className={styles.summary}>
-          <li>
+          <li className={styles['summary-item']}>
             <h4 className={styles.heading}>
               {storagePair ? 'Account from the storage:' : 'Randomly generated account:'}
             </h4>
-
+            <div className={styles.separator} />
             {pair && (
               <div className={styles.account}>
-                <Identicon value={pair.address} theme="polkadot" size={14} />
-                <span>{getVaraAddress(pair.address)}</span>
+                <span className={styles.value}>{shortenString(getVaraAddress(pair.address), 4)}</span>
+                <FileCopyIcon
```

Comment:
should be `<Button icon={CopySVG} color='transparent' />`, we shouldn't assign `onClick` and other interactive actions on non-interactive elemets